llama_stack/models/llama/quantize_impls.py

```diff
@@ -6,9 +6,10 @@
 
 # type: ignore
 import collections
-import logging
 
-log = logging.getLogger(__name__)
+from llama_stack.log import get_logger
+
+log = get_logger(name=__name__, category="llama")
```

Comment:
Shouldn't it be "models" instead of "llama" ?

Reply:
i actually prefer your proposal at https://github.com/meta-llama/llama-stack/pull/3065#issuecomment-3175744860

```
Use model::<model name> for per-model code (e.g., model::llama3).
```

changing accordingly 👍🏽